### PR TITLE
add sentinel-golang example

### PR DESCRIPTION
**Pull request by pantianying**
add dubbo-go use sentinel-golang demo

### CODE REVIEW

File: golang/filter/sentinel/dubbo/go-server/app/server.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+	_ "github.com/apache/dubbo-go/common/proxy/proxy_factory"
+	_ "github.com/apache/dubbo-go/filter/filter_impl"
+
+	_ "github.com/apache/dubbo-go/cluster/cluster_impl"
+	_ "github.com/apache/dubbo-go/cluster/loadbalance"
+	_ "github.com/apache/dubbo-go/registry/zookeeper"
+)
+
+import (
```
Comment (AlexStocks) [inline golang/filter/sentinel/dubbo/go-server/app/server.go]: hey, guy, this import block should be placed on the second place.

File: golang/filter/sentinel/dubbo/go-client/app/version.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+var (
+	Version = "2.6.0"
```
Comment (AlexStocks) [inline golang/filter/sentinel/dubbo/go-client/app/version.go]: 2.7.5

File: golang/filter/sentinel/dubbo/go-client/app/client.go [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+			})
+
+			// The program exits normally or timeout forcibly exits.
+			fmt.Println("app exit now...")
+			return
+		}
+	}
+}
+
+func println(format string, args ...interface{}) {
```
Comment (AlexStocks) [inline golang/filter/sentinel/dubbo/go-client/app/client.go:117]: pls use https://github.com/dubbogo/gost/blob/master/log/color.go#L107 instead.

File: golang/filter/sentinel/dubbo/go-server/app/user.go [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+func (u *UserProvider) Reference() string {
+	return "UserProvider"
+}
+
+func (u User) JavaClassName() string {
+	return "com.ikurento.user.User"
+}
+
+func println(format string, args ...interface{}) {
```
Comment (AlexStocks) [inline golang/filter/sentinel/dubbo/go-server/app/user.go:62]: use https://github.com/dubbogo/gost/blob/master/log/color.go#L103 instead

File: golang/filter/sentinel/dubbo/go-client/app/client.go [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+ */
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
```
Comment (wuzhidexiaolang) [inline golang/filter/sentinel/dubbo/go-client/app/client.go:25]: 1111

**Pull request closed without merging**